tests/unit_tests/kotahi_docmaps/v1/test_data.py

```diff
@@ -88,14 +88,17 @@
     'manuscript_versions': [MANUSCRIPT_VERSION_1, MANUSCRIPT_VERSION_2]
 }
 
-REVIEW_1 = '''
-    Reviewer #1 (Public Review):
+SPACE_1 = ' '
+COLON_1 = ' '
```

Comment:
It says colon, but is it? :-)

Do you think it is more readable with a colon constant?

Reply:
ah yes :D